src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -790,6 +790,76 @@ static char[] Resize(char[] buffer)
+
+            using (var cn = new Microsoft.Data.SqlClient.SqlConnection(DataTestUtility.TCPConnectionString))
+            {
+                await cn.OpenAsync();
+
+                using (var cmd = cn.CreateCommand())
+                {
+                    cmd.CommandText = $"""
+                   if exists (select *
+                                  from sysobjects
+                                  where name = 'Expl_User_Global_Sets_'
```

Comment:
The table name needs to be randomized or it will stomp of other parallel test runs. (Mainly an issue when the tests target Azure SQL DB.) It also needs to be dropped at the end.